src/Apps/Artwork/Utils/createCollectUrl.tsx

```diff
@@ -0,0 +1,46 @@
+import { data as sd } from "sharify"
+
+export const createCollectUrl = (
```

Comment:
How about refactoring this to use `"qs"`? It's already in our bundle, and it's a bit more maintainable than string concatenation :)

Here's an example usage: https://github.com/ds300/reaction/blob/f51af697e0871022d8754e558b82826e10b5a50b/src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAboutTheWorkFromPartner.tsx#L65

Reply:
Wow I had no idea that was a thing! Will do!